docs/logging/README.md

```diff
@@ -0,0 +1,130 @@
+# Logging stack
+
+### Why do we need logging stack?
+We need logging stack because currently Kubernetes use the underlying container runtime logging which does not persist log for not existing containers. This makes difficult to investigate issues ocurred in long ago destroyed containers.
```

Comment:
```suggestion
We need logging stack because currently Kubernetes use the underlying container runtime logging which does not persist log for not existing containers. This makes it difficult to investigate issues occurred in long ago destroyed containers.
```